frontend/components/TooltipWrapper/TooltipWrapper.tsx

```diff
@@ -14,7 +14,7 @@ const baseClass = "component__tooltip-wrapper";
 const TooltipWrapper = ({
   children,
   tipContent,
-  position = "bottom",
+  position = "top",
```

Comment:
This is causing a bug seen here:
 
<img width="348" alt="Screenshot 2022-11-22 at 2 30 10 PM" src="https://user-images.githubusercontent.com/71795832/203404995-ce55d508-d77a-4baa-af33-db083a70f775.png">